content/blog/2023/06/22/2023-06-22-mirrors-jenkins-new-IP.adoc

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "New Public IPv4 for Jenkins Mirrors"
+tags:
+- infrastructure
+- mirrors
+- jenkins
+authors:
+- dduportal
+opengraph:
+  image: /images/post-images/2023/01/12/jenkins-newsletter/infrastructure.png
+links:
+  discourse: https://community.jenkins.io/t/new-public-ipv4-for-jenkins-mirrors/8137
+---
+
+The Jenkins project provides a download mirror infrastructure allowing to download Jenkins packages and plugins from a download server close to your location.
+
+It provides a "HTTP redirector" service hosted behind the `get.jenkins.io`, `mirrors.jenkins.io` and `mirrors.jenkins-ci.org` domains, with a new public IP (`20.119.232.75`) since the 12th of June 2023.
+The former redirector service and its previous IPv4 will be decommissioned the 27 of June 2023.
+
+[IMPORTANT]
+====
+Please update your DNS servers and firewall rules to the new IP `20.119.232.75` if you have a restricted environment.
```

Comment:
```suggestion
IMPORTANT: Please update your DNS servers and firewall rules to the new IP `20.119.232.75` if you are in a restricted environment.
```
Reading the doc from your link we don't even need blocks as there are only single phrases.

(Also changed the verb)

Reply:
I have a preference on using the block because the string `IMPORTANT:` does not really make it explicit that an admonition is used, but it is a detail